lib/cluster.js

```diff
@@ -350,7 +350,7 @@ function masterInit() {
     }
   }
 
-  cluster.fork = function(env) {
+  cluster.fork = (env) => {
```

Comment:
ok , will change it back , in the other cases like this one, 
i.e cluster.funcname = () = {}
instead of cluster.funcname = function(){}
would i be wiser to change it back as well ?
due to the case mentioned in the issue comments by silverwind here:
https://github.com/nodejs/node/issues/8913#issuecomment-255869643